CHANGELOG.md

```diff
@@ -1,468 +1,517 @@
 # Changelog
 
+## 1.28.0 - 2023-05-18
+
+- Feature: runScript command now support conditional execution
+- Feature: Improved debug output:
+  - Shows failure reason when command fails
+  - Generates screenshot when command fails
+  - Unified most logs under ~/.maestro/tests/<date>/maestro.log
+- Feature: Launch arguments support for long values
+- Tweak: JUnit report naming changes. Local and Cloud should now have the same naming convention.
+- Tweak: Added deprecation notice for experimental features
+- Fix: maestro record command was not working on iOS
+- Fix: WebDriver, only scroll to elements outside of the window befiore tapping
+- Fix: close request leaking body
+- Fix: maestro cloud now will fail on timeout if configured as such
```

Comment:
This is not clear to me 🤔

Reply:
I believe previously failOnTimeout was always false. Now failOnTimeout mimics failOnCancellation
https://github.com/mobile-dev-inc/maestro/commit/5fb056326eb0134d63197abd24a74810c191b9aa